src/main/java/it/pagopa/transactions/commands/handlers/TransactionSendClosureHandler.java

```diff
@@ -99,28 +115,71 @@ public Mono<TransactionClosureSentEvent> handle(TransactionClosureSendCommand co
                                 );
 
                         TransactionClosureSentEvent event = new TransactionClosureSentEvent(
-                                transaction.getTransactionId().value().toString(),
-                                transaction.getRptId().value(),
-                                transaction.getTransactionActivatedData().getPaymentToken(),
+                                tx.getTransactionId().value().toString(),
+                                tx.getRptId().value(),
+                                tx.getTransactionActivatedData().getPaymentToken(),
                                 closureSendData
                         );
 
-                        return transactionEventStoreRepository.save(event);
-                    });
-        }
-    }
+                                Mono<NotificationEmailResponseDto> emailResponse = notificationsServiceClient.sendSuccessEmail(
+                                        new NotificationsServiceClient.SuccessTemplateRequest(
+                                                tx.getEmail().value(),
+                                                "Hai pagato un avviso di pagamento PagoPA",
```

Comment:
Hi @giovanniberti ,
is this subject valid when closePayment returns with status ko?

Reply:
Nope, but this PR is concerned with the success case only